migrations/2_deploy_dharma_contracts.js

```diff
@@ -1,6 +1,4 @@
-const MULTISIG_SIGNATORIES = require("./MULTISIG_SIGNATORIES");
-
-const LIVE_NETWORK_ID = "live";
+const CONSTANTS = require("./migration_constants");
```

Comment:
We use ES6 style imports all throughout the rest of the codebase.  We should do the same here, if possible. (i.e. `import * as CONSTANTS from "./migration_constants"`)